exercises/matrices/matrix-1d-cpp/test/golden-model-py/golden_model.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import json
+
+
+def get_determinant(m):
+    try:
+        return np.linalg.det(m)
```

Comment:
Does this still resolves determinant with a not 100% accurate value?

Reply:
No, it's used as protection to prevent the code from crashing.
The determinant issue is "resolved" by rounding the result to it's nearest integer value.